packages-native/intro-screen/package.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "intro-screen",
+    "widgetName": "IntroScreen",
+    "version": "1.0.0",
+    "license": "Apache-2.0",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/mendix/widgets-resources.git"
+    },
+    "config": {
+        "widgetPath": "../test-project/mxproject/widgets",
+        "projectPath": "../test-project/mxproject",
+        "subprojectFolder": "packages-native/intro-screen"
+    },
+    "packagePath": "com.mendix.widget.native",
+    "scripts": {
+        "start": "../../node_modules/.bin/pluggable-widgets-tools start:ts:native --subprojectPath $npm_package_config_subprojectFolder",
+        "build": "../../node_modules/.bin/pluggable-widgets-tools build:ts:native --subprojectPath $npm_package_config_subprojectFolder",
+        "build:release": "../../node_modules/.bin/pluggable-widgets-tools release:ts:native --subprojectPath $npm_package_config_subprojectFolder",
+        "lint": "../../node_modules/.bin/eslint --config ../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/",
+        "test": "../../node_modules/.bin/jest --config ../../scripts/test/jest.native.config.js",
+        "version": "npm run build:release",
+        "postversion": "../../node_modules/.bin/ts-node ../../scripts/release/UpdateGithubRelease.ts --project ../../scripts/tsconfig.json"
+    },
+    "devDependencies": {
+        "@types/react-native-app-intro-slider": "^3.0.0"
+    },
+    "dependencies": {
+        "@native-mobile-resources/util-widgets": "^0.0.1"
```

Comment:
Should we bump the version of the utils to 1.0.0, as we are using it for production builds. Or do we still miss something?

Reply:
True, just bumped